package.json

```diff
@@ -85,6 +85,19 @@
       "@changesets/get-github-info@0.6.0": "patches/@changesets__get-github-info@0.6.0.patch",
       "@changesets/assemble-release-plan@6.0.3": "patches/@changesets__assemble-release-plan@6.0.3.patch",
       "vitest-websocket-mock": "patches/vitest-websocket-mock.patch"
-    }
+    },
+    "ignoredBuiltDependencies": [
+      "cpu-features",
+      "esbuild",
+      "lmdb",
+      "msgpackr-extract",
+      "msw",
+      "protobufjs",
+      "ssh2",
+      "workerd"
+    ],
```

Comment:
This is a new feature coming from `pnpm` v10. It doesn't look like we need any of these to have their install scripts (build native deps) run.